procedures/release-duty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -36,141 +25,47 @@ builds (3 weeks before the release), gives enough time to everybody.
-| //bugzilla.mozilla.org/sh | mozilla-esr52                            |
-| ow_bug.cgi?id=1339076>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1339832 <https:          | Buildbot specific changes to enable      |
-| //bugzilla.mozilla.org/sh | mozilla-esr52                            |
-| ow_bug.cgi?id=1339832>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1340669 <https:          | Update merge day and esr docs            |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1340669>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1341330 <https:          | Please add mozilla-esr52 to DXR          |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1341330>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1341491 <https:          | Set ESR_NEXT in ship-it config           |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1341491>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1342117 <https:          | Set watch_all_branches to True for ESR52 |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1342117>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1342204 <https:          | Enable TC on mozilla-esr52               |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1342204>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1342431 <https:          | Turn off TC-based Android jobs on ESR52  |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1342431>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1343097 <https:          | Add mozilla-esr52 and comm-esr52 to      |
-| //bugzilla.mozilla.org/sh | release-notifications heroku app         |
-| ow_bug.cgi?id=1343097>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1343366 <https:          | browser_parsable_css.js fails on ESR52   |
-| //bugzilla.mozilla.org/sh | ASAN builds due to the ESR branding      |
-| ow_bug.cgi?id=1343366>`__ | changes                                  |
-+---------------------------+------------------------------------------+
+for the `tracking bug <https://bugzilla.mozilla.org/show_bug.cgi?id=1717527>`__
```

Comment:
This looks complete to me but CC'ing @jcristau in case there's anything else he hit too